udemy_enroller/udemy.py

```diff
@@ -68,6 +68,8 @@ def __init__(self, settings: Settings, cookie_file_name: str = ".cookie"):
         self._all_course_ids = []
         self._currency_symbol = None
         self._currency = None
+        self.counter_enroled: int = 0
+        self.counter_already_enroled: int = 0
```

Comment:
These aren't used anymore. 
You can probably reset this file as I don't think there were any changes made to it